tools/db_bench_tool.cc

```diff
@@ -617,8 +617,11 @@ DEFINE_int64(simcache_size, -1,
 DEFINE_bool(cache_index_and_filter_blocks, false,
             "Cache index/filter blocks in block cache.");
 
+DEFINE_bool(use_cache_jemalloc_no_dump_allocator, false,
+            "Use JemallocNodumpAllocator for block/blob cache.");
+
 DEFINE_bool(use_cache_memkind_kmem_allocator, false,
-            "Use memkind kmem allocator for block cache.");
+            "Use memkind kmem allocator for block/blob cache.");
 
```

Comment:
Why not use the MemoryAllocator::CreateFromString instead?  This way you would not have to introduce new booleans for every allocator and check for incompatibilities and precedence?  You could also use/set any options on the allocator that they expose and the user wanted?